README.md

```diff
@@ -425,7 +425,7 @@ introduction section, but a good abstract will be shorter, less detailed, and
 broader in scope than the introduction. Simply copying and pasting the first
 few paragraphs of the introduction is tempting, but it generally results in an
 abstract that is both incomplete and redundant. An abstract will typically be
-five to ten lines. An abstract of more than 20 lines or fewer than three lines
+around 50-150 words in length. An abstract that is much shorter or longer
```

Comment:
words are better than lines. However, I think we should put the limits as recommendation not restriction. I would suggest : As abstract will typically be around 50-150 words in length. Depending on content and subject of the document this bound can be relaxed, however, authors are highly recommended to within the word bounds mentioned here.

Reply:
It is a recommendation?

Reply:
yes :-). If this makes sense.